packages/host/app/components/matrix/user-profile.gts

```diff
@@ -48,9 +56,16 @@ export default class UserProfile extends Component {
     </div>
     <div class='button-container'>
       {{#if this.isEditMode}}
+        <Button
+          class='user-button'
+          data-test-profile-canel-btn
```

Comment:
typo?

Reply:
My keyboard is starting to flake out, about 10% of my keypresses aren’t registered 😤